docs/_includes/header.html

```diff
@@ -2,6 +2,9 @@
 <nav class="navbar" >
     <div class="container">
         <div class="navbar-brand">
+            {% for logo in page.header %}
+            <img src="{{ logo.logo_image }}" height="90" width="90">
```

Comment:
I think all you need is the width attribute. If you'd prefer to have both attributes, I would set height to "auto"

Reply:
I actually tried both. didn't work for me. could you try this on your side too?

Reply:
worked now. thanks

Reply:
done